cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -870,6 +870,7 @@ void DecodeDeltaByteArray(cudf::detail::hostdevice_span<PageInfo> pages,
                           size_t num_rows,
                           size_t min_row,
                           int level_type_size,
+                          size_t* initial_str_offsets,
```

Comment:
Just curious, would it be better to use a `span` here?

Reply:
Yeah I think that would be better.